api/authorization-api-1_0.md

```diff
@@ -476,449 +471,74 @@ X-Request-ID: bfe9eb29-ab87-4ca3-be83-a1d5d8305716
-
-#### Subject Query Result {#subject-query-result}
-A Subject Query Result is array of Subject Query Decisions ({{example-subject-query-decision}}). It is JSON object combining a subject, a list of resource attribute names and an action. Given that a Subject Query result is expected to be the response to a Subject Search, only positive matches should be returned; i.e., only those subjects that match the search criteria (those subjects that are allowed to access the provided Resource Attributes). Any Subjects absent from the results do not have any access to the Resource. 
-A Subject Query Result has the following fields:
-
-actions:
-: OPTIONAL. An Array of the action for which the decision is provided. The format is as described in the Actions section ({{actions}}). The values in this list should match the values provided as queries in the Subject Search request.
-
-attributeNames:
-: OPTIONAL. An Array of attribute names of the resource for which the response applies. The attribute is provided only if attributes were part of the Subject search request. In that case, the attribute names must match those that are part of the request.
-
-subject:
-: REQUIRED. The subject for which the decision is provided. The format is as described in the Subjects section ({{subjects}}). 
-
-The following is a non-normative example of a Subject Query Decision:
-
-~~~ json
-{
-  "actions": ["delete", "read", "write"],
-  "attributeNames": [
-    "author",
-    "createDate",
-    "lastUpdated"
-  ],
-  "subject": {
-    "id": "alex@3edges.com"
-  }
-}
-~~~
-{: #example-subject-query-decision title="Example Subject Query Decision"}
-
-Following is a non-normative example of a Subject Search Response:
-
-~~~ http
-HTTP/1.1 OK
-Content-type: application/json
-X-Request-ID: bfe9eb29-ab87-4ca3-be83-a1d5d8305720
-
-{
-  "iat": 1234567890,
-  "resource": {
-    "id": "somevalue",
-    "type": "document",
-    "attributeNames": [
-      "author",
-      "createDate",
-      "lastUpdated"
-    ]
-  },
-  "queries": [
-    "write",
-    "read"
-  ],
-  "decisions": [
-    {
-      "action": "write",
-      "attributeNames": [
-        "author"
-      ],
-      "subject": {
-        "id": "alex@3edges.com"
-      }
-    },
-    {
-      "action": "read",
-      "attributeNames": [
-        "author",
-        "createDate",
-        "lastUpdated"
-      ],
-      "subject": {
-        "id": "alex@3edges.com"
-      }
-    },
-    {
-      "action": "read",
-      "attributeNames": [
-        "author",
-        "createDate",
-        "lastUpdated"
-      ],
-      "subject": {
-        "id": "Janet@3edges.com"
-      }
-    }
-  ],
-  "nextPageToken": "1DlR0Em5panAPy5llasLPfNUpDztEKgTDKF2I5gPwymnc"
-}
-~~~
-{: #example-subject-search-response title="Example of a Subject Search Response"}
-
-# Transport
+The following errors are indicated by the status codes defined below:
 
-## REST Binding
+| Code | Description  | HTTPS Body Content |
```

Comment:
We had a chat a while back that some proxies/gateways cannot inspect the response payload and rely on HTTP codes to determine access. Do we want to have a flag that allows the PDP to return HTTP 403 in case of decision false?

Reply:
I agree that if we add this, we would want to control it with a flag.

Reply:
discussed during the call - agreed to leave this out of scope for now (and certainly for this PR).